lib/nexpose/site.rb

```diff
@@ -488,6 +487,10 @@ def to_h
           organization: @organization.to_h,
           users: users
       }
+      # Only pass in blackouts if they were actually specified
```

Comment:
If we're going to release these changes let's update this comment and add an `@note`/`@todo` comment about needing this for backwards compatibility with Nexpose.